README.md

```diff
@@ -7,6 +7,7 @@
   * [configure_software](#configure_software)
   * [install_software](#install_software)
   * [install_python3](#install_python3)
+  * [Multiple Aurora Installations on one device](#multiple-aurora-installations-on-one-device)
```

Comment:
I think it makes sense to have this as introduction in How to run section. If you leave it like that it looks like another example